.github/workflows/upgrade-python-requirements.yml

```diff
@@ -0,0 +1,55 @@
+name: Upgrade Requirements
+
+on:
+  schedule:
+    # will start the job at 01:30 UTC every Wednesday
+    - cron: "30 1 * * 3"
```

Comment:
When would be a good time for this bot to run so that we can respond to those PRs?

Reply:
which time do you think might be good? I always check for PRs in the morning so that time might be good for me.

Reply:
I think we should change it a bit from what other repos use so that we are not creating peaks of trafic for the ci repo or anything.
How about we change it to Tuesdays and make it run at 1:15 UTC?

Reply:
Good!